autoload/AnsiEsc.vim

```diff
@@ -2111,11 +2111,8 @@ fun! s:MultiElementHandler()
       continue
      elseif skip == 385
       " handling <esc>[38;5;...
-      if has("gui") && has("gui_running")
-       let fg= s:Ansi2Gui(code)
-      else
-       let fg= code
-      endif
+      let fg= code
+      let guifg= s:Ansi2Gui(code)
```

Comment:
Probably you should define both new vars in function header, just like existing vars were defined.